exonum-light-client/src/main/java/com/exonum/client/request/BlockTimeOption.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exonum.client.request;
+
+import com.exonum.client.response.Block;
+
+/**
+ * Request option for block commit time.
+ * See {@linkplain Block#getCommitTime()}.
+ */
+public enum BlockTimeOption {
+  /**
+   * Identifies to do not include block commit times in a response.
```

Comment:
I am not sure 'identify' works here. @Maria-Nosyk wdyt? 

---

Here it might be fine to skip it completely, saying "Do not include block commit times in a response".